### PR TITLE
Link to demo videos #150

### LINKED ISSUE #150: Add Oct 2022 demo video to README
Matrix Public Archive is awesome but it wasn't on my radar until I heard about it on [Matrix Live](https://matrix.org/blog/2022/08/05/this-week-in-matrix-2022-08-05#matrix-public-archive-website): https://www.youtube.com/watch?v=UT6KSEqDUf8&t=548s

Should we link to this video from the README? I find it absolutely inspiring! It works! It's coming!

Screen Shot 2022-12-03 at 12 59 38 PM

### PR DESCRIPTION
**Pull request by pdurbin**
Link to demo videos and TWIM blog posts that describe show-off the project

Fix #150

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ the messages from a Matrix homeserver.
 Re-using Hydrogen gets us pretty and native(to Element) looking styles and keeps
 the maintenance burden of supporting more event types in Hydrogen.
 
+## Demo videos
+
+- [Aug 2022](https://www.youtube.com/watch?v=6KHQSeJTXm0&t=583s) ([blog post](https://matrix.org/blog/2022/08/05/this-week-in-matrix-2022-08-05#matrix-public-archive-website))
+- [Oct 2022](https://www.youtube.com/watch?v=UT6KSEqDUf8&t=548s)
+
```
Comment (MadLittleMods) [inline README.md:35]: We can move this section below the screenshot (move up one section) to group the showcase pieces together

File: README.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ the messages from a Matrix homeserver.
 Re-using Hydrogen gets us pretty and native(to Element) looking styles and keeps
 the maintenance burden of supporting more event types in Hydrogen.
 
+## Demo videos
+
+- [Aug 2022](https://www.youtube.com/watch?v=6KHQSeJTXm0&t=583s) ([blog post](https://matrix.org/blog/2022/08/05/this-week-in-matrix-2022-08-05#matrix-public-archive-website))
```
Comment (MadLittleMods) [inline README.md:33]:
```suggestion
- [Aug 2022](https://www.youtube.com/watch?v=6KHQSeJTXm0&t=583s) ([blog post](https://matrix.org/blog/2022/08/05/this-week-in-matrix-2022-08-05#matrix-public-archive-website)): A quick intro of what the project looks like, the goals, and what it accomplishes. The new portal into the Matrix ecosystem 🌌
```

File: README.md [hunk 3]
```diff
@@ -28,6 +28,11 @@ the messages from a Matrix homeserver.
...
+- [Oct 2022](https://www.youtube.com/watch?v=UT6KSEqDUf8&t=548s)
```
Comment (MadLittleMods) [inline README.md:34]:
```suggestion
- [Oct 2022](https://www.youtube.com/watch?v=UT6KSEqDUf8&t=548s): Showing off the room directory landing page used to browse everything available in the archive
```

File: README.md (hunk 2 above)
Comment (MadLittleMods) [reply README.md:33]: We could also add a little YouTube logo to spice up these links a little bit (applies to both)

```suggestion
- [![](https://user-images.githubusercontent.com/558581/206083768-d18456de-caa3-463f-a891-96eed8054686.png) Aug 2022](https://www.youtube.com/watch?v=6KHQSeJTXm0&t=583s) ([blog post](https://matrix.org/blog/2022/08/05/this-week-in-matrix-2022-08-05#matrix-public-archive-website)): A quick intro of what the project looks like, the goals, and what it accomplishes. The new portal into the Matrix ecosystem 🌌
```

File: README.md (hunk 1 above)
Comment (pdurbin) [reply README.md]: Sure. Moved in 77859e9.

File: README.md (hunk 3 above)
Comment (pdurbin) [reply README.md]: Added in 77859e9.

**Comment by pdurbin** [comment]
@MadLittleMods thanks for the review. I took another swing in 77859e9.

**Comment by MadLittleMods** [comment]
@pdurbin Thanks for the contribution to show off the project better 🐠

**Pull request merged**